cdk/lib/support-workers.ts

```diff
@@ -27,15 +27,36 @@ import {
 } from "aws-cdk-lib/aws-stepfunctions";
 import { LambdaInvoke } from "aws-cdk-lib/aws-stepfunctions-tasks";
 
-type PaymentProvider = "Stripe" | "DirectDebit" | "PayPal";
+const Contribution = "Contribution";
```

Comment:
It would be good to share these types with the frontend at some point

Reply:
💯% that'd be great